src/veins/base/messages/AirFrame.msg

```diff
@@ -1,12 +1,16 @@
 cplusplus {{
```

Comment:
These changes appear to be unused in the rest of the patches. Are they necessary?